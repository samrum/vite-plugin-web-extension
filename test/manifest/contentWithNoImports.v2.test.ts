import { getResourceDir, runManifestV2Test } from "./manifestTestUtils";

const resourceDir = getResourceDir("contentWithNoImports");

runManifestV2Test("contentWithNoImports", () => ({
  content_scripts: [
    {
      js: [`${resourceDir}/content.js`],
      matches: ["https://*/*", "http://*/*"],
    },
  ],
}));
