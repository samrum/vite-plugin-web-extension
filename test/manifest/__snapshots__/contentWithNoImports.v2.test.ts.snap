// Vitest Snapshot v1

exports[`contentWithNoImports - Manifest V2 1`] = `
[
  {
    "fileName": "test/manifest/resources/contentWithNoImports/content.js",
    "name": undefined,
    "source": "console.log(\\"content\\");
",
    "type": "asset",
  },
  {
    "fileName": "manifest.json",
    "name": undefined,
    "source": "{
  \\"version\\": \\"1.0.0\\",
  \\"name\\": \\"Manifest Name\\",
  \\"manifest_version\\": 2,
  \\"content_scripts\\": [
    {
      \\"js\\": [
        \\"test/manifest/resources/contentWithNoImports/content.js\\"
      ],
      \\"matches\\": [
        \\"https://*/*\\",
        \\"http://*/*\\"
      ]
    }
  ]
}",
    "type": "asset",
  },
]
`;
